gems/aws-sigv4/spec/spec_helper.rb

```diff
@@ -1,11 +1,6 @@
 # frozen_string_literal: true
 
 require 'rspec'
-require 'simplecov'
```

Comment:
What happened to simplecov? I don't think sigv4 should share from core if that's what's happening. This is stand alone.

Reply:
It gets started twice because this file requires the core spec_helper.  

The coverage is still standalone.

Reply:
This is a consequenece of running the tests together.  We can only start simple cov once.

Alternatives - create seperate targets for core and sigv4 and never run the tests at the same time.

Reply:
Yeah I think we should. It's a separate gem, which core will depend on.